### PR TITLE
priors_fix: Fixed issue when creating priors egs 

### PR DESCRIPTION
**Pull request by vimalmanohar**
Fixed issue when creating priors egs in get_egs_discrimiative2.sh

### CODE REVIEW

**Comment by danpovey** [comment]
This setup is rather wasteful because you copy the transforms and alignments to disk, but you then use them only once.  There is no point dumping something if you only need it once.  In both cases you can just give it an archive containing everything, and it will take only what it needs.

**Comment by vimalmanohar** [comment]
I have modified it. If it is ok, I can make a clean patch.

**Comment by danpovey** [comment]
Look OK.  Sure, make a clean patch.

**Comment by vimalmanohar** [comment]
Made a clean patch.

**Comment by danpovey** [comment]
Thanks.  @vijayaditya, could you please do a quick pass of checking before I merge this..

**Comment by vijayaditya** [comment]
OK

File: egs/wsj/s5/steps/nnet2/get_egs_discriminative2.sh [hunk 1]
```diff
@@ -256,39 +255,23 @@ if [ $stage -le 10 ]; then
 priors_egs_list=
 for y in `seq $num_archives_priors`; do
```
Comment (vijayaditya) [inline egs/wsj/s5/steps/nnet2/get_egs_discriminative2.sh]: Why is num_archives_priors still being used ? I thought the entire point was to just have one job for prior computation.

Comment (danpovey) [inline egs/wsj/s5/steps/nnet2/get_egs_discriminative2.sh]: It's OK-- creating the egs for the priors is fast but doing the computation
using them is not that fast.  So I didn't propose to have just one archive
of priors.
Dan

**Comment by vijayaditya** [comment]
OK, in that case looks fine to merge.

**Pull request merged**